src/commands/key/sign.ts

```diff
@@ -0,0 +1,226 @@
+import { Command, Flags } from "@oclif/core";
```

Comment:
could we move the file to like `entrypoint/approve.ts` or something? i know i'm definitely going to browse the code at some point and think this is a generic sign

Reply:
I can make this a top level approve on the same level as execute. The token permit is also necessary for calls to operators and Projects contracts as well for depositing stake and escrow respectively

Reply:
Although looking at those deposit methods they seem to have just coupled the signature into the contract call itself which is fine. Can create an `entrypoint` subdir and move the `createWithEntrypoint` command over there as well

Reply:
your call. initially i just thought the name and dir was a bit confusing so wanted to throw out a suggestion but feel free to do what you think is easiest to understand